### PR TITLE
RHDEVDOCS-4583 Document changes in installation and upgrade channels

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.11`, `enterprise-4.12`
- **JIRA issues**: [RHDEVDOCS-4583 Document changes in installation and upgrade channels](https://issues.redhat.com/browse/RHDEVDOCS-4583)
- **Preview pages**: 
  - [Installation page](http://file.pnq.redhat.com/sosarkar/4583-operator-channels/cicd/pipelines/installing-pipelines.html#op-installing-pipelines-operator-in-web-console_installing-pipelines): See the bullet point "Select an Update Channel." in the **Procedure**.
  - [1.8 RN](http://file.pnq.redhat.com/sosarkar/4583-operator-channels/cicd/pipelines/op-release-notes.html#deprecated-features-1-8_op-release-notes)
- **SME + QE review**: @VeereshAradhya

### CODE REVIEW

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 1]
```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
+*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.8` is `pipelines-1.8`.
```
Comment (VeereshAradhya) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:44]: Now the default channel is `latest` (https://issues.redhat.com/browse/SRVKP-2412)
cc; @vdemeester

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 2]
```diff
@@ -41,12 +41,12 @@ The supported profiles are:
...
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing and upgrading the {pipelines-title} Operator.
+*** To install a specific version of the {pipelines-title} Operator, cluster adminsitrators can also the `pipelines-<version>` channel. For example, to install the {pipelines-title} Operator version `1.8`, you can use the `pipelines-1.8` channel.
 +
 [NOTE]
 ====
-The `preview` and `stable` channels will be deprecated and removed in a future release.
+The `preview` and `stable` are not available for installation and upgrade of the Operator.
```
Comment (VeereshAradhya) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:49]: In 4.9 and 4.10, the `stable` and `preview` channels are still present. It is not present in 4.11 and above.
cc: @concaf @ppitonak @Vincent056 
Screenshot from 2022-11-07 15-40-08

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 3]
```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing and upgrading the {pipelines-title} Operator.
+*** To install a specific version of the {pipelines-title} Operator, cluster adminsitrators can also the `pipelines-<version>` channel. For example, to install the {pipelines-title} Operator version `1.8`, you can use the `pipelines-1.8` channel.
```
Comment (VeereshAradhya) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:45]: Operator version `1.8.x`?

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 4]
```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing and upgrading the {pipelines-title} Operator.
```
Comment (VeereshAradhya) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:44]: Do not know if we need to specify `latest` is the default channel for upgrading.
@ppitonak WDYT?

File: modules/op-release-notes-1-8.adoc [hunk 5]
```diff
@@ -313,6 +313,13 @@ $ oc get route -n openshift-pipelines pipelines-as-code-controller \
 [id="deprecated-features-1-8_{context}"]
 == Deprecated and removed features
 
+* Starting with {product-title} 4.11, the `preview` and `stable` channels for installing and upgrading the {pipelines-title} Operator is removed. To install and upgrade the Operator, use the appropriate `pipelines-<version>` channel, or the `latest` channel for the most recent stable version. For example, to install the {pipelines-shortname} Operator version `1.8`, use the `pipelines-1.8` channel.
```
Comment (VeereshAradhya) [inline modules/op-release-notes-1-8.adoc:316]: `1.8.x`?

**Review by VeereshAradhya — APPROVED** [review]

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 6]
```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing the {pipelines-title} Operator.
+*** To install a specific version of the {pipelines-title} Operator, cluster adminsitrators can also the `pipelines-<version>` channel. For example, to install the {pipelines-title} Operator version `1.8.x`, you can use the `pipelines-1.8` channel.
```
Comment (ppitonak) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:45]: 1. typo adminsitrators
2. can also ... some verb missing ... the `pipelines-<version>` channel

**Review by ppitonak — APPROVED** [review]

**Review by jc-berger — APPROVED** [review]
/lgtm

File: modules/op-release-notes-1-8.adoc [hunk 7]
```diff
@@ -313,6 +313,13 @@ $ oc get route -n openshift-pipelines pipelines-as-code-controller \
 [id="deprecated-features-1-8_{context}"]
 == Deprecated and removed features
 
+* Starting with {product-title} 4.11, the `preview` and `stable` channels for installing and upgrading the {pipelines-title} Operator is removed. To install and upgrade the Operator, use the appropriate `pipelines-<version>` channel, or the `latest` channel for the most recent stable version. For example, to install the {pipelines-shortname} Operator version `1.8.x`, use the `pipelines-1.8` channel.
```
Comment (rolfedh) [inline modules/op-release-notes-1-8.adoc:316]:
```suggestion
* Starting with {product-title} 4.11, the `preview` and `stable` channels for installing and upgrading the {pipelines-title} Operator are removed. To install and upgrade the Operator, use the appropriate `pipelines-<version>` channel, or the `latest` channel for the most recent stable version. For example, to install the {pipelines-shortname} Operator version `1.8.x`, use the `pipelines-1.8` channel. 
```

**Review by rolfedh — COMMENTED** [review]
Hi Souvik. Would you fix this minor grammatical error?

**Pull request merged**